test/tools/ossfuzz/SolidityGenerator.h

```diff
@@ -190,8 +194,6 @@ class SolidityGenerator: public std::enable_shared_from_this<SolidityGenerator>
 public:
 	explicit SolidityGenerator(unsigned _seed);
 
-	/// Returns a multi-source test case.
-	std::string visit();
```

Comment:
Interesting, this change wasn't caught early on. Maybe there is a compiler flag to detect this?

Reply:
Somehow, I'm not able to find a flag that warns on declared but undefined (and unused) functions. I tried `-Wunused` and `-Wall` as a last resort with clang-10 :(